### PR TITLE
 Bug fix in the read/write/delete samples with LocalStack

### LINKED ISSUE #2052: when running the QuickStart readflow, AWS CLI doesn't accept "theshire" as a Region
when running manually the QuickStart, when you get to the part where you need to[ register your asset to localstack](https://fybrik.io/v1.3/samples/notebook-read/#prepare-a-dataset-to-be-accessed-by-the-notebook:~:text=For%20experimentation%20you,%24%7BFILEPATH%7D), 
configuring REGION as "theshire" produces an error - 
 
 "An error occurred (InternalError) when calling the PutObject operation (reached max retries: 2): exception while calling s3.PutObject: 'theshire' is not a valid AWS region name for s3"

I am using aws-cli/2.7.21

local stack was installed using the commands specified in the read flow

### PR DESCRIPTION
**Pull request by aradhalevy**
Added control over the version of LocalStack that is installed (not just taking the latest) as part of the samples flow.

This way, when LocalStack updates it won't break like happened #2052.

closes #2052

### CODE REVIEW

File: samples/OneClickDemo/OneClickDemo-OMD.sh [hunk 1]
```diff
@@ -25,6 +27,7 @@ if [[ "$FYBRIK_VERSION" == "" ]]; then
   # Get Fybrik lateset realease from github
   FYBRIK_VERSION=$(git -c 'versionsort.suffix=-' ls-remote --tags --sort='v:refname' https://github.com/fybrik/fybrik.git | tail --lines=1 | cut --delimiter='/' --fields=3)
 fi
+FYBRIK_BRANCH_OMD="$FYBRIK_VERSION"
```
Comment (roytman) [inline samples/OneClickDemo/OneClickDemo-OMD.sh:30]: why do you need a separate `FYBRIK_BRANCH_OMD` env variable?

File: site/docs/samples/notebook-read.md [hunk 2]
```diff
@@ -36,13 +36,17 @@ Make a note of the service endpoint, bucket name, and access credentials. You wi
     === "Kubernetes"
             helm repo add localstack-charts https://localstack.github.io/helm-charts
             helm install localstack localstack-charts/localstack \
+                 --version {{LOCALSTACK_CHART_VERSION}} \
+                 --set image.tag="{{LOCALSTACK_VERSION}}" \
                  --set startServices="s3" \
                  --set service.type=ClusterIP \
-                 --set livenessProbe.initialDelaySeconds=25
+                 --set livenessProbe.initialDelaySeconds=25
```
Comment (roytman) [inline site/docs/samples/notebook-read.md:43]: pls remove the `space` at the end of the line

File: samples/OneClickDemo/OneClickDemo-OMD.sh [hunk 3]
```diff
@@ -259,7 +261,7 @@ bin/kubectl config set-context --current --namespace=fybrik-notebook-sample
 
 header "\nInstall localstack"
 bin/helm repo add localstack-charts https://localstack.github.io/helm-charts
-bin/helm install localstack localstack-charts/localstack --version 0.4.3 --set startServices="s3" --set service.type=ClusterIP --set image.tag="1.2.0" --set livenessProbe.initialDelaySeconds=25
+bin/helm install localstack localstack-charts/localstack --version ${LOCALSTACK_CHART_VERSION} --set startServices="s3" --set service.type=ClusterIP  --set livenessProbe.initialDelaySeconds=25 --set image.tag="${LOCALSTACK_VERSION}"
```
Comment (roytman) [inline samples/OneClickDemo/OneClickDemo-OMD.sh:264]: please remove a redundant space between  `service.type=ClusterIP  --set `

**Review by roytman — COMMENTED** [review]
Thanks for the PR, several tiny comments.

File: samples/OneClickDemo/OneClickDemo-OMD.sh (hunk 1 above)
Comment (aradhalevy) [reply samples/OneClickDemo/OneClickDemo-OMD.sh]: it is as it is installed in the [QuickStart](https://fybrik.io/v1.3/get-started/quickstart/#install-data-catalog), it is used for the [OpenMetaData script](https://github.com/fybrik/fybrik/blob/master/third_party/openmetadata/install_OM.sh#L9), but it needs to be FYBRIK_BRANCH and exported for it to work for that.
I changed it accordingly.

File: site/docs/samples/notebook-read.md (hunk 2 above)
Comment (aradhalevy) [reply site/docs/samples/notebook-read.md]: Done

File: samples/OneClickDemo/OneClickDemo-OMD.sh (hunk 3 above)
Comment (aradhalevy) [reply samples/OneClickDemo/OneClickDemo-OMD.sh]: Done

**Review by roytman — APPROVED** [review]

**Comment by BrodaUa** [comment]
Hello @aradhalevy @roytman 

as of version v1.3.2, this fix is missing in OneClickDemo, see:
https://raw.githubusercontent.com/fybrik/fybrik/v1.3.2/samples/OneClickDemo/OneClickDemo-OMD.sh

+ the same issue appears in the step-by-step guides page here:
https://fybrik.io/v1.3/samples/notebook-read/ (localstack setup does not work - complaints about 'theshire' region)

**Pull request merged**